### ISSUE TITLE
Copyright agent fails to show copyrights without license information

### ISSUE THREAD

**Issue by rlintu** [fossology/fossology#508]
Luckily the correction is easy. 

--- a/src/copyright/ui/ajax-copyright-hist.php
+++ b/src/copyright/ui/ajax-copyright-hist.php
@@ -170,7 +170,7 @@ class CopyrightHistogramProcessPost extends FO_Plugin
 
     $join = "";
     $filterQuery = "";
-    if ($type == 'statement' && $filter == "nolics")
+    if ($type == 'statement' && $filter == "nolic")
     {
       $noLicStr = "No_license_found";
       $voidLicStr = "Void";

Maybe someone can patch the code without pull request.

**Comment by rlintu** [comment]
More readable patch
```
--- a/src/copyright/ui/ajax-copyright-hist.php
+++ b/src/copyright/ui/ajax-copyright-hist.php
@@ -170,7 +170,7 @@ class CopyrightHistogramProcessPost extends FO_Plugin
 
     $join = "";
     $filterQuery = "";
-    if ($type == 'statement' && $filter == "nolics")
+    if ($type == 'statement' && $filter == "nolic")
     {
       $noLicStr = "No_license_found";
       $voidLicStr = "Void";
```

**Comment by steffen-weber** [comment]
The wrong filter value is also in src/copyright/ui/list.php:174.

@rlintu can you create a pull request instead of a patch?

**Comment by rlintu** [comment]
I'll do that.

**Comment by steffen-weber** [comment]
fixed in cca937a73e4f942842ffe54e885b2ff3700af203
